Integrations/integration-CyberTriage.yml

```diff
@@ -180,25 +182,31 @@ script:
         is_true = lambda x: x == 'yes'
         is_hash_upload_on = is_true(CMD_ARGS['malware_hash_upload']) # arg value = 'yes' or 'no'
         is_file_upload_on = is_true(CMD_ARGS['malware_file_upload']) # arg value = 'yes' or 'no'
-        is_fast_scan_on = not(is_true(CMD_ARGS['full_scan']))        # arg value = 'yes' or 'no'
         endpoint = CMD_ARGS['endpoint']
+        scan_options = CMD_ARGS['scan_options']
```

Comment:
Please add `scan_options` to the command arguments in integration settings, and please use `.get()` so you won't get Key Error exception in case the user did not pass it.

Reply:
scan_options has a default value in Demisto so .get() was not needed.

Reply:
I see, well `scan_options` is not defined in integration settings, so couldn't see it has default.